### DISCUSSION TITLE
How to create a custom starter file on v5?

### DISCUSSION THREAD

**Discussion by BatuhanK** [adonisjs/core#2541] [Help]
Hello, I wanted to create simple scripts which uses Adonisjs without http server.

in v4, we created a file like that

`starter.js`

```js
const { Ignitor } = require('@adonisjs/ignitor')


new Ignitor(require('@adonisjs/fold'))
  .appRoot(__dirname)
  .preLoad(`./app/${process.argv[2]}.js`)
  .fire()

```

and create sample script on `./app/Tasks/Sample.js` like below

```js
const User = use('App/Models/User')

async function main () {
  const user = await User.find(1)
  console.log(user)
}

main()

```

Then we can run to invoke our sample script
```bash
node starter Tasks/Sample
```

How can I scaffold a starter file like that in v5?

**Comment by thetutlage** [comment]
Why not create an ace command?

**Reply by BatuhanK** [reply]
Commands has boilerplate code and doesnt have top level requires. If possible, I prefer this but if not I will create command to mimic my starter.js
